src/components/PurchaseOrder/Summary/SummaryForm.js

```diff
@@ -30,7 +30,7 @@ const SummaryForm = ({ initialValues: order }) => (
       <FieldIsApproved disabled={Boolean(order.workflowStatus) && !isWorkflowStatusIsPending(order)} />
     </Col>
     <Col xs={6} md={3}>
-      <FieldWorkflowStatus isNonInteractive={order?.workflowStatus} />
+      <FieldWorkflowStatus isNonInteractive />
```

Comment:
Do we really need this field? At all times it should be displayed as text

Reply:
but it was already implemented, why not?